src/cli/global/install.rs

```diff
@@ -61,6 +61,10 @@ pub struct Args {
     /// Specifies that the packages should be reinstalled even if they are already installed.
     #[arg(action, long)]
     force_reinstall: bool,
+
+    /// Specify that the menu items of the environment should be installed, defaults to true.
+    #[arg(long, default_value_t = true)]
+    menu_install: bool,
```

Comment:
```suggestion
    /// Specify that the menu items of the environment should be installed, defaults to true.
    #[arg(long)]
    no_menu_install: bool,
```

Boolean flags should always default to false, otherwise they can't be disabled